Lib/pdb.py

```diff
@@ -231,6 +231,9 @@ def __init__(self, completekey='tab', stdin=None, stdout=None, skip=None,
             pass
         self.allow_kbdint = False
         self.nosigint = nosigint
+        # Consider these characters as part of the command so when the users type
+        # c.a or c['a'], it won't be recognized as a c(ontinue) command
+        self.identchars = cmd.Cmd.identchars + '=.[]()"\'+-*/&|<>~^'
```

Comment:
Shouldn't modulo (%) and matrix multiplication (@) be covered as well?

Reply:
You are right, I also added `,`.